server/src/grpc/mod.rs

```diff
@@ -68,51 +153,24 @@ pub async fn grpc_server() {
 /// Tokio signal handler that will wait for a user to press CTRL+C.
 /// We use this in our tonic `Server` method `serve_with_shutdown`.
 ///
-/// # Arguments
-///
-/// # Examples
-///
 /// ```
 /// tonic::transport::Server::builder()
 ///     .serve_with_shutdown(&"0.0.0.0:50051".parse().unwrap(), shutdown_signal())
 ///     .await?;
 /// ```
-
 pub async fn shutdown_signal() {
     tokio::signal::ctrl_c()
         .await
         .expect("Tokio signal ctrl-c received!");
     grpc_info!("signal shutdown!");
 }
 
-///Implementation of gRPC endpoints
-#[derive(Debug, Default, Copy, Clone)]
-pub struct StorageImpl {}
-
-#[tonic::async_trait]
-impl StorageRpc for StorageImpl {
-    /// Returns ready:true when service is available
-    async fn is_ready(
-        &self,
-        _request: Request<ReadyRequest>,
-    ) -> Result<Response<ReadyResponse>, Status> {
-        let response = ReadyResponse { ready: true };
-        Ok(Response::new(response))
-    }
-}
-
-impl From<ArrErr> for Status {
-    fn from(err: ArrErr) -> Self {
-        // These errors come from modules like Postgres, where you
-        // probably wouldn't want to include error details in the
-        // response, log them here instead which will include
-        // tracing information from the request handler
-        //
-        // <https://cheatsheetseries.owasp.org/cheatsheets/REST_Security_Cheat_Sheet.html#error-handling>
-        // <https://cheatsheetseries.owasp.org/cheatsheets/Logging_Cheat_Sheet.html#which-events-to-log>
-        let err: Error = err.into();
-        grpc_warn!("{:#}", err);
-
-        tonic::Status::internal("error".to_string())
+pub fn get_runtime_handle() -> Handle {
```

Comment:
Please explain the purpose of the runtime handle in a comment, seeing that it's used to async get the PSQL pool without blocking the main executor

Reply:
fixed, added docs incl. example usage